### PR TITLE
Pushups part 2A

### PR DESCRIPTION
**Pull request by Misseswaddles**

### CODE REVIEW

File: students/C_Farris/session01/break_me.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#Author: Carol Farris
+#File Name: break_me.py
+#Content: Write four simple python functions that throw errors
+
+
+#The function below will throw a name error if supplied an undefined value
+def cause_name_error(val):
+    try:
+        print(val)
+    except NameError:
```
Comment (PythonCHB) [inline students/C_Farris/session01/break_me.py]: did you actually get this to raise? right now, "val" is defined?

And we're not expecting you to catch these error yet, just wanted you to experience seeing them.

File: students/C_Farris/session01/break_me.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+def cause_type_error(val):
+	try: 
+		print(val//2)
+	except TypeError:
+		print("TypeError! The variable passed is invalid for the desired action.")	
+
+#The function below will throw a syntax error because the print statement
+#was incorrectly written. A perenthesis needs to surround ("hello")
+#It needs to be commented out or the script wont run.
+"""def cause_syntax_error():
```
Comment (PythonCHB) [inline students/C_Farris/session01/break_me.py]: that'll do it.

File: students/C_Farris/session02/grid_printer.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+#Date: October 16, 2018
+#Created by: Carol Farris
+#Purpose: Practice printing a grid
+
+
+#Below successfully prints a grid, but isn't complete.
+#It will print a total of n boxes, in a grid width of 2
+#I need to make it more flexible.
```
Comment (PythonCHB) [inline students/C_Farris/session02/grid_printer.py]: but a fine start.

File: students/C_Farris/session03/fizz_buzz.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+#Date: October 23, 2018
+#Created by: Carol Farris
+#Purpose: Practice Fizz Buzz challenge
+
+
+
+#Method that prints Fizz if number x is divisible 3 and Buzz if number x is divisible by 5
+#This method tests 1-100 inclusive. 
+def fizzbuzz():
```
Comment (PythonCHB) [inline students/C_Farris/session03/fizz_buzz.py]: you got it!

**Comment by Misseswaddles** [comment]
updated final grid printer exercise.

**Pull request merged**